### PR TITLE
For code review.

### PR DESCRIPTION
**Pull request by hjeeves**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -90,7 +94,7 @@ idea {
             logger.info('Running browser tests on ' + project.intTest_selenium_server_url)
         }
         else {
-            systemProperty 'selenium.server.url', 'http://cadcint2.cadc.dao.nrc.ca:3444'
+            systemProperty 'selenium.server.url', 'http://cadcint2.cadc.dao.nrc.ca:4444'
```
Comment (at88mph) [inline build.gradle]: Leave this at port 3444, which is the Selenium 3 server.

File: build.gradle [hunk 2]
```diff
@@ -38,8 +41,9 @@ ext {
     docker_image_name = 'mach378.cadc.dao.nrc.ca:5000/citation'
 
     // Change this value to test other locations, such as Production or RC.
-    intTest_default_web_app_url = 'http://rcapps.canfar.net'
-    intTest_default_web_app_endpoint = '/citation'
+//    intTest_default_web_app_url = 'http://rcapps.canfar.net'
+    intTest_default_web_app_url = 'http://jeevesh.cadc.dao.nrc.ca'
```
Comment (at88mph) [inline build.gradle]: Default should still be rcapps.canfar.net, and the default endpoint should now be `/citation`.

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+<c:if test="${empty baseURL}">
+  <c:set var="req" value="${pageContext.request}" />
+  <c:set var="url">${req.requestURL}</c:set>
+  <c:set var="uri" value="${req.requestURI}" />
+  <c:set var="baseURL" value="${fn:substring(url, 0, fn:length(url) - fn:length(uri))}" />
+</c:if>
+
+<%-- Set this by configuration in the future. --%>
+<%--<c:set var="resourceCapabilitiesEndPoint" value="http://apps.canfar.net/reg/resource-caps" />--%>
+<c:set var="resourceCapabilitiesEndPoint" value="http://jeevesh.cadc.dao.nrc.ca/reg/resource-caps" />
```
Comment (at88mph) [inline src/main/webapp/index.jsp]: This should be the Production URL.

File: src/main/webapp/js/citation_request.js [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+            page.setProgressBar('error')
+            page.setAjaxFail(message)
+          })
+      })
+      return false
+    }
+
+    // TODO: hook this in as a secondary call after the initial GET
+    // returns successfully...
+    //function getDoiStatus(doiName) {
```
Comment (at88mph) [inline src/main/webapp/js/citation_request.js]: Is this unfinished?  Unnecessary?  If it's unnecessary, please remove it.

File: src/main/webapp/js/citation_request.js [hunk 5]
```diff
@@ -0,0 +1,467 @@
...
+        doiName +
+        '/data' +
+        '" target="_blank">/storage/list/' +
+        astrodataDir +
+        doiName +
+        '/data</a>'
+      $('#doi_data_dir').html(dataUrl)
+
+      // Once the Mint function is completed, this will be displayed
+      //var landingPageClose = ".html?view=data";
```
Comment (at88mph) [inline src/main/webapp/js/citation_request.js]: Is this commented out stuff unfinished? Unnecessary? If it's unnecessary, please remove it.

File: src/main/webapp/request/index.jsp [hunk 6]
```diff
@@ -13,7 +13,8 @@
 </c:if>
 
 <%-- Set this by configuration in the future. --%>
-<c:set var="resourceCapabilitiesEndPoint" value="http://apps.canfar.net/reg/resource-caps" />
+<%--<c:set var="resourceCapabilitiesEndPoint" value="http://apps.canfar.net/reg/resource-caps" />--%>
+<c:set var="resourceCapabilitiesEndPoint" value="http://jeevesh.cadc.dao.nrc.ca/reg/resource-caps" />
```
Comment (at88mph) [inline src/main/webapp/request/index.jsp]: This should be a Production URL.

File: build.gradle (hunk 1 above)
Comment (hjeeves) [reply build.gradle]: ok

File: build.gradle (hunk 2 above)
Comment (hjeeves) [reply build.gradle]: ok

File: src/main/webapp/index.jsp (hunk 3 above)
Comment (hjeeves) [reply src/main/webapp/index.jsp]: applies to both webapp/request/index.jsp and webapp/index.jsp...  okay.

File: src/main/webapp/js/citation_request.js (hunk 5 above)
Comment (hjeeves) [reply src/main/webapp/js/citation_request.js]: It was unfinished. Just caught it this morning.  I've updated it in one of the latest checkins.

**Pull request merged**